hazelcast/src/main/java/com/hazelcast/nio/serialization/FieldKind.java

```diff
@@ -35,7 +35,13 @@
     ARRAY_OF_BOOLEAN(1),
     INT8(2),
     ARRAY_OF_INT8(3),
+    /**
```

Comment:
Minor: I would put a newline before that to make it more clear that this is the Javadoc for the item below. I know it is a well known convention, but assuming this will be read by other non-Java developers, I feel like it would be better.